src/entities/CheckScenes/task/taskRunnerSqs.ts

```diff
@@ -43,6 +49,53 @@ const placesAttributes: Array<keyof PlaceAttributes> = [
   "textsearch",
 ]
 
+const ACCESS_KEY = env("AWS_ACCESS_KEY")
+const ACCESS_SECRET = env("AWS_ACCESS_SECRET")
+const BUCKET_HOSTNAME = env("BUCKET_HOSTNAME")
+const BUCKET_NAME = env("AWS_BUCKET_NAME", "")
+
+async function updateGenesisCityManifest() {
```

Comment:
Can we move this function to a different file?

Reply:
Sure thing!